src/wallet.cpp

```diff
@@ -3443,7 +3448,7 @@ void CWallet::AutoZeromint()
         return;
     }
 
-    CAmount nZerocoinBalance = GetZerocoinBalance();
+    CAmount nZerocoinBalance = GetPendingZerocoinBalance();
```

Comment:
Shouldn't this be 
CAmount nZerocoinBalance = GetZerocoinBalance(false);
Or else we are going to compute the percentage of minting zerocoin instead of minted + minting (while before it was only minted).

Reply:
Yes, you are definitely correct. Great catch, and thank you very much!

Reply:
You are welcome. Be careful, there might be the same thing a few lines down.